docs/using_requests_locally.md

```diff
@@ -0,0 +1,105 @@
+# Using Cachito requests locally
```

Comment:
@chmeliik do we want to include a link to the file into the main README file?

Reply:
Good idea! Will do.